flytekit/core/base_task.py

```diff
@@ -463,7 +463,7 @@ def dispatch_execute(
         new_user_params = self.pre_execute(ctx.user_space_params)
         from flytekit.deck.deck import _output_deck
 
-        new_user_params._decks = [ctx.user_space_params.default_deck]
```

Comment:
does this change the user interface?  If we don't have one yet, can we get a cookbook example that shows the usage of a flyte deck.

Reply:
no, it doesn't change the interface. we append a new deck to `new_user_params` in deck.py, so we don't need to it again here. https://github.com/flyteorg/flytekit/blob/29237baa7a5a057af16776911e9dea42617864a9/flytekit/deck/deck.py#L63

cookbook example: https://github.com/flyteorg/flytesnacks/pull/735